uxarray/_scrip.py

```diff
@@ -1,11 +1,121 @@
-from warnings import warn
+import xarray as xr
+import numpy as np
 
 
-def _read_scrip(filepath):
-    """This is just a function placeholder.
+def _to_ugrid(in_ds, outfile):
```

Comment:
Arg name "outfile" does not make sense as `_to_ugrid(ext_ds, ds)` on Line 98 gets in a dataset, `ds`, and populates it.

Reply:
Would it be better to just use the same variable names from when this is called in `_read_scrip` in this case?